pkg/cri/server/services.go

```diff
@@ -481,3 +482,20 @@ func (s *server) Status(ctx context.Context,
 
 	return rsp.(*criv1.StatusResponse), err
 }
+
+func (s *server) CheckpointContainer(ctx context.Context, req *criv1.CheckpointContainerRequest) (*criv1.CheckpointContainerResponse, error) {
+	rsp, err := s.interceptRequest(ctx, runtimeService, checkpointContainer, req,
+		func(ctx context.Context, req interface{}) (interface{}, error) {
+			return (*s.runtime).CheckpointContainer(ctx, req.(*criv1.CheckpointContainerRequest))
+		})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return rsp.(*criv1.CheckpointContainerResponse), err
+}
+
+func (s *server) GetContainerEvents(req *criv1.GetEventsRequest, src criv1.RuntimeService_GetContainerEventsServer) error {
+	return nil
```

Comment:
Hmm, I was wondering would the right thing to do be returning a "not implemented" error?

Reply:
It depends whether we want to mimick `containerd` or `cri-o` ;-) But seriously... it might be better to just return an error. I'll update it like that.